stix_shifter_modules/alertflex/entry_point.py

```diff
@@ -0,0 +1,54 @@
+from stix_shifter_utils.utils.base_entry_point import BaseEntryPoint
```

Comment:
change this line to `from stix_shifter_utils.utils.entry_point_base import EntryPointBase`

Reply:
After my commit 52335a1, Alertflex adapter now is able to work with stix-shifter-develop branch, but unfortunately, it has stopped work for stix-shifter-master branch.  After implementing your two suggested changes Alertflex module stopped work for both branches. Please, propose a solution.